frontend/src/components/TableComponents/TableCell.tsx

```diff
@@ -10,21 +10,27 @@ type TableCellProps = {
   last: boolean;
   tag?: string;
   buttonHandler?: () => void;
+  ButtonModal?: () => JSX.Element;
 }
 
 const TableCell = (props: TableCellProps) => {
-  const { data, index, first, last, tag, buttonHandler } = props;
+  const { data, index, first, last, tag, buttonHandler, ButtonModal } = props;
   if (first) {
     return (<><td
       key={index}
       className={cn(styles.passInfo, styles.cell, styles.firstCell)}>
       {data}
     </td></>);
   } if (last) {
+    if (buttonHandler && ButtonModal) {
+      return (<><td key={index} className={`${styles.passInfo} ${styles.cell} 
```

Comment:
<> shouldn't be necessary since whole element has a single outer layer, <td>

Reply:
Whoops, I just copy pasted from the other rows which had <>. Will remove it from those